cmd/juju/backups/create.go

```diff
@@ -20,16 +20,30 @@ import (
 const (
 	notset          = backups.FilenamePrefix + "<date>-<time>.tar.gz"
 	downloadWarning = "downloading backup archives is recommended; " +
-		"backups stored remotely are not guaranteed to be available"
+		"backups stored remotely are not guaranteed to be available."
 )
 
 const createDoc = `
-create-backup requests that Juju creates a backup of its state and prints the
+This command requests that Juju creates a backup of its state and prints the
 backup's unique ID.  You may provide a note to associate with the backup.
 
-The backup archive and associated metadata are stored remotely by Juju, but
-will also be copied locally unless --no-download is supplied. To access the
-remote backups, see 'juju download-backup'.
+By default, the backup archive and associated metadata are downloaded without keeping a copy remotely on the controller.
+
+Use --no-download to avoid getting a local copy of the backup downloaded at the end of the backup process. 
+
+Use --keep-copy option to store a copy of this backup remotely on the controller. Ignored when --no-download is used.
```

Comment:
I'd prefer it to complain instead of ignoring. The user intent may have been to --keep-copy and they typed --no-download by mistake. We should be strict about accepting valid CLI args.

Reply:
Nice :+1: 
added a complain :D